.changeset/four-readers-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': patch
+---
+
+**fix**(`SSelect`): add option groups with headers and select all buttons
```

Comment:
It seems that it is a significant feature, and definitely not a fix.

Why not `feature`? Why `patch` and not `minor`?

Reply:
It was in 'documentation' but not implemented. I fixed it